### PR TITLE
Wait child process during cluster setup on packet

### PR DESCRIPTION
**Pull request by lobkovilya**
Signed-off-by: Ilya Lobkov <ilya.lobkov@xored.com>

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .cloudtest/packet/start-master.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 KUBERNETES_INIT_VERSION=1.16.3
-kubeadm init --kubernetes-version "${KUBERNETES_INIT_VERSION}" --pod-network-cidr=192.168.0.0/16 --skip-token-print
+kubeadm init --kubernetes-version "${KUBERNETES_INIT_VERSION}" --pod-network-cidr=192.168.0.0/16 --skip-token-print || exit 1
```
Comment (alex-yust) [inline .cloudtest/packet/start-master.sh]:
```suggestion
kubeadm init --kubernetes-version "${KUBERNETES_INIT_VERSION}" --pod-network-cidr=192.168.0.0/16 --skip-token-print || exit $?
```

File: .cloudtest/packet/create-kubernetes-cluster.sh [hunk 2]
```diff
@@ -21,9 +21,22 @@ wait
...
 ssh ${SSH_OPTS} root@${worker_ip} ./download-worker-images.sh &
-wait
+pids+=" $!"
+
+for pid in $pids; do
+  echo "waiting for PID " "$pid"
+  wait "$pid"
+  exitcode=$?
+  if [ "$exitcode" != 0 ]; then
+    echo "node setup failed" && exit 9
```
Comment (alex-yust) [inline .cloudtest/packet/create-kubernetes-cluster.sh]:
```suggestion
    echo "node setup failed: process exited with code $exitcode, aborting.." && exit 9
```

Comment (denis-tingajkin) [inline .cloudtest/packet/create-kubernetes-cluster.sh]: should we also throw `exitcode` instead of `9`?

Comment (lobkovilya) [reply .cloudtest/packet/create-kubernetes-cluster.sh]: I think no, because there is own exit code for each line :)

**Comment by haiodo** [comment]
failed tests are not related to Packet startup scripts so merging.

**Pull request merged**